dashboard/test/controllers/api/v1/teacher_feedbacks_controller_test.rb

```diff
@@ -296,10 +295,6 @@ def parsed_response
     JSON.parse(@response.body)
   end
 
-  def formatted_response
```

Comment:
I think I changed `TeacherFeedbacksController#count` to return the number without quotes. Please let me know if I need to preserve the quotes.